### PR TITLE
Street names with data from OSM and Wikidata

### LINKED ISSUE #44: Visualize gender equality in street names
Similar to https://github.com/EqualStreetNames/equalstreetnames/ (see e.g. https://munich.equalstreetnames.eu/), visualize the streets that are named after persons and calculate the percentage of male and female eponyms.

They use OpenStreetMap data and its [`name:etymology:wikidata` tag](https://wiki.openstreetmap.org/wiki/Key:name:etymology:wikidata).

### PR DESCRIPTION
**Pull request by fxedel**
Fixes #44 (and visualizes other data, too).

### CODE REVIEW

**Comment by fxedel** [comment]
@FloEdelmann wanna review?

File: R/strassennamen.R [hunk 1]
```diff
@@ -0,0 +1,229 @@
...
+          ), height = 550) %>%
+            addProviderTiles(providers$CartoDB.Positron) %>%
+            addPolylines(
+              data = strassenNamensherkuenfte$Geometry,
+              color = unname(typColors[strassenNamensherkuenfte$Typ]),
+              label = paste0(strassenNamensherkuenfte$Name, ": ", strassenNamensherkuenfte$Typ)
+            ) %>%
+            identity()
+        }
+      )
```
[the remark this answers is not in the document]
Comment (fxedel) [reply R/strassennamen.R:181]: Ah, almost forgot about that one. I suggest extracting this into another issue, same with street length and other possible street metrics.

File: R/strassennamen.R [hunk 2]
```diff
@@ -0,0 +1,229 @@
...
+strassenNamensherkuenfte <- left_join(
+  osmStrassen %>% separate_longer_delim(NamensherkunftWikidata, "/"),
+  wikidataNamensherkuenfte,
+  by = join_by(NamensherkunftWikidata == WikidataObjekt)
+) %>% mutate(
+  Typ = coalesce(Typ, "Unbekannt")
+) %>% mutate(
+  Typ = factor(Typ, levels = c(
+    "Personen mit Lokalbezug",
+    "Komponisten",
```
[the remark this answers is not in the document]
Comment (fxedel) [reply R/strassennamen.R:44]: I also thought about that. I think I don't wanna have it in the data due the colon character, but it's fine for the UI.

**Comment by fxedel** [comment]
@FloEdelmann thanks for the review!

**Comment by FloEdelmann** [comment]
Thanks for implementing this!

File: R/strassennamen.R (hunk 1 above)
[the remark this answers is not in the document]
Comment (FloEdelmann) [reply R/strassennamen.R:185]: I opened https://github.com/fxedel/vaterstetten-in-zahlen/issues/51

**Pull request merged**